pandas/io/parsers.py

```diff
@@ -167,9 +168,13 @@
     to preserve and not interpret dtype.
     If converters are specified, they will be applied INSTEAD
     of dtype conversion.
-engine : {{'c', 'python'}}, optional
-    Parser engine to use. The C engine is faster while the python engine is
-    currently more feature-complete.
+engine : {{'c', 'python', 'pyarrow'}}, optional
+    Parser engine to use. The C and pyarrow engines are faster, while the python engine
+    is currently more feature-complete. The pyarrow engine requires ``pyarrow`` > 0.15
```

Comment:
is this > 0.15? better just say >= 0.15.1 (or 0.16.0)

Reply:
sorry typo. meant to say >=0.15.0